### PR TITLE
Setup CRUD for game model and added SimpleForm

### PR DESCRIPTION
**Pull request by dannypoit**
- Ran rails scaffold to create MVC for 'game'
- Added 'name' column to game table
- Added simple_form gem

### CODE REVIEW

**Review by mscoutermarsh — APPROVED** [review]
Nice!

**Comment by mscoutermarsh** [comment]
For the merge, will need to resolve conflicts.

For the build - I'm guessing Rubocop will have a couple errors with this changes. You can run `rubocop -a` to autocorrect.

**Comment by mscoutermarsh** [comment]
Also - looks like your branch doesn't have the travis config yet. I'd `rebase` from master to get the latest changes.

File: spec/controllers/games_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  }
+
+  # This should return the minimal set of values that should be in the session
+  # in order to pass any filters (e.g. authentication) defined in
+  # GamesController. Be sure to keep this updated too.
+  let(:valid_session) { {} }
+
+  describe "GET #index" do
+    it "returns a success response" do
+      game = Game.create! valid_attributes
```
Comment (mscoutermarsh) [inline spec/controllers/games_controller_spec.rb]: change this to `Game.create! valid_attributes`

the `game` variable isn't used

File: spec/controllers/games_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      get :index, params: {}, session: valid_session
+      expect(response).to be_success
+    end
+  end
+
+  describe "GET #show" do
+    it "returns a success response" do
+      game = Game.create! valid_attributes
+      get :show, params: {id: game.to_param}, session: valid_session
```
Comment (mscoutermarsh) [inline spec/controllers/games_controller_spec.rb]: It works here because `game` is used

**Comment by dannypoit** [comment]
Mike rebased into game-crud-new, and I merged that. This can be closed.

**Pull request closed without merging**